overlord/hookstate/ctlcmd/set.go

```diff
@@ -60,6 +72,23 @@ func (s *setCommand) Execute(args []string) error {
 		return fmt.Errorf("cannot set without a context")
 	}
 
+	var snapAndPlugOrSlot snap.SnapAndName
```

Comment:
This is a close counterpart to the get implementation above, so many (most?) of the commands are relevant here as well.